devops/resources/run-ansible.sh

```diff
@@ -44,5 +44,13 @@ if [[ -n "${BECOME_USER}" ]]; then
   ANSIBLE_BECOME=" --become --become-user=${BECOME_USER} "
 fi
 
+#SOME_THING_THAT_CHECKS_FOR_FLAG_FILE
+# only run docker shim command if 1) flag file exists 2) the shim args exist
+if [[ SOME_CONDITIONAL_HERE && -n "${ANSIBLE_MODULE_ARGS_SHIM}" ]]; then
```

Comment:
NIT If the flag file exists but the shim commands do not, go ahead and blow up so it will be obvious we forgot to define the shim